include/boost/geometry/algorithms/detail/relate/point_point.hpp

```diff
@@ -223,7 +222,8 @@ struct multipoint_multipoint
     {
         // sort points from the 1 MPt
         typedef typename geometry::point_type<SortedMultiPoint>::type point_type;
-        typedef geometry::less<void, -1, CSTag> less_type;
+        //TODO: pass strategy
+        typedef geometry::less<void, -1, Strategy> less_type;
```

Comment:
I'm a bit worried that `geometry::less` is changed. This really looks like a breaking change to me...

So it fixes convex hull (good!) but it needs also changes in many other operations like `relate` and `is_valid` etc...

Reply:
For everything apart from convex hull the same strategy is called. E.g. for cartesian it is `cartesian<ComparePolicy, EqualsPolicy, -1>` which is as the old one https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/strategies/compare.hpp#L165 (also called from default compare strategy) with the only difference that now it has the EqualsPolicy parameter which for all algorithms is as before (i.e. `strategy::compare::equals_epsilon` see `strategies/relate/cartesian.hpp` but for convex hull it is `strategy::compare::equals_exact` see `strategies/convex_hull/cartesian.hpp`). This change eventually change only convex hull strategies.

Reply:
But `less` it is a public policy, meant for all library users. It’s not internal. So that we don’t use it like this does not mean that we can change it.

Reply:
This is updated and now the user can pass either a strategy or a cs tag as the third parameter to `less` (the second choice can be considered as deprecated and removed in the future). So in some sense now `less` is "enhanced". What do you think?

Reply:
sound good!